components/invitation_modal/invitation_modal_members_step.test.jsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 
 import {shallowWithIntl} from 'tests/helpers/intl-test-helper';
 
-import InvitationModalMembersStep from './invitation_modal_members_step.jsx';
+import InvitationModalMembersStep from './invitation_modal_members_step/invitation_modal_members_step.jsx';
```

Comment:
You might want to move this test file into the folder you've created so that everything is organized nicely.